manifests/init.pp

```diff
@@ -205,6 +205,10 @@
 # @param private_key
 #   Path to the key associated with this node's certificate.
 #
+# @param private_key_type
+#   The type of the private key. Usually the private key is of type RSA key but it can also be an Elliptic Curve key (EC) or DSA.
+#   Valid values: rsa, dsa, ec
```

Comment:
Data type will be shown in the generated REFERENCE.md, so no need to repeat it here.
```suggestion
```

Reply:
ok, i suppress this line and i rebase my branch